src/main/config/config.c

```diff
@@ -1056,4 +1058,3 @@ uint32_t featureMask(void)
 {
     return masterConfig.enabledFeatures;
 }
-
```

Comment:
Any reason to remove this newline? No newline at the end of the file violates POSIX and C standards.